frontend/src/components/MovieList.tsx

```diff
@@ -0,0 +1,39 @@
+import {Movie} from "../Movie.ts";
+import styled from "styled-components";
+
+type MovieListProps = {
+    movies: Movie[]
+    title: string
+}
+
+export default function MovieList(props: MovieListProps) {
+
+    const posterImageUrl: string = "https://image.tmdb.org/t/p/w440_and_h660_face"
+
+    return (
+        <>
+            <h1>{props.title}</h1>
+            <StyledContainerSection>
+                {props.movies.map((movie: Movie) => (
+                    <StyledArticle key={movie.id}><img src={posterImageUrl + movie.poster_path}
+                                                       alt={"Poster of " + movie.title}/>
+                        <section>
+                            <p>{movie.title}</p>
```

Comment:
Maybe change title in movies, because movie.title and props.title it irritating.